elixir/apps/domain/lib/domain/mailer/sync_email/sync_error.html.eex

```diff
@@ -0,0 +1,136 @@
+		<meta name="format-detection" content="telephone=no, date=no, address=no, email=no, url=no">
+		<meta name="color-scheme" content="light dark">
+		<meta name="supported-color-schemes" content="light dark">
+		<!--[if mso]>
+		<noscript>
+			<xml>
+				<o:OfficeDocumentSettings xmlns:o="urn:schemas-microsoft-com:office:office">
+					<o:PixelsPerInch>96</o:PixelsPerInch>
+				</o:OfficeDocumentSettings>
+			</xml>
+		</noscript>
+		<style>
+			td,th,div,p,a,h1,h2,h3,h4,h5,h6 {font-family: "Segoe UI", sans-serif; mso-line-height-rule: exactly;}
+		</style>
+		<![endif]-->
+		<title>Firezone Sync Error</title>
+		<style>
+			.hover-important-text-decoration-underline:hover {
+				text-decoration: underline !important
+			}
+			:is([dir="rtl"] .rtl-text-right) {
+				text-align: right !important
+			}
+			@media (max-width: 600px) {
+				.sm-my-8 {
+					margin-top: 32px !important;
+					margin-bottom: 32px !important
+				}
+				.sm-px-4 {
+					padding-left: 16px !important;
+					padding-right: 16px !important
+				}
+				.sm-px-6 {
+					padding-left: 24px !important;
+					padding-right: 24px !important
+				}
+				.sm-leading-8 {
+					line-height: 32px !important
+				}
+			}
+		</style>
+	</head>
+	<body style="margin: 0; width: 100%; background-color: #f6f6f6; padding: 0; -webkit-font-smoothing: antialiased; word-break: break-word">
+		<div role="article" aria-roledescription="email" aria-label="Firezone Sync Error" lang="en">
+			<div class="sm-px-4" style="background-color: #f6f6f6; font-family: ui-sans-serif, system-ui, -apple-system, 'Segoe UI', sans-serif">
+				<table align="center" cellpadding="0" cellspacing="0" role="none">
+					<tr>
+						<td style="width: 552px; max-width: 100%">
+							<div class="sm-my-8" style="margin-top: 48px; margin-bottom: 48px; text-align: center">
+								<a href="https://firezone.dev">
+									<div>
+										<img src="https://www.firezone.dev/images/logo-lockup.png" width="250" alt="Firezone logo" style="max-width: 100%; vertical-align: middle; line-height: 1">
+									</div>
+								</a>
+							</div>
+							<table style="width: 100%;" cellpadding="0" cellspacing="0" role="none">
+								<tr>
+									<td class="sm-px-6" style="border-radius: 4px; background-color: #ffffff; padding: 48px; font-size: 16px; color: #4f4f4f; box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.05)">
+										<h1 class="sm-leading-8" style="margin: 0 0 24px; font-size: 24px; font-weight: 600; color: #000000">
+											Identity Provider Sync Error!
+										</h1>
+										<p style="margin: 0; line-height: 24px">
+											An Identity Provider setup in Firezone has failed to sync <%= @provider.last_syncs_failed %> times.
+										</p>
+										<div role="separator" style="line-height: 16px">&zwj;</div>
+										<p style="margin: 0; line-height: 24px;">
+											Below is the last sync error message:
+											<div>
+												<%= @provider.last_sync_error %>
+											</div>
```

Comment:
IIRC this error could potentially be long. What do you think of using `<code>` blocks for it? It might help readability too if it contains JSON or other monospace-appropriate text.

Reply:
Good catch.  I meant to go back and put that in before I pushed the PR up but forgot.  Will get it updated.

Reply:
Here's a new screenshot:

<img width="693" alt="Screenshot 2024-08-30 at 1 11 22 AM" src="https://github.com/user-attachments/assets/c7850f4d-b292-4b11-a1e3-d57b10f2a74d">